src/components/tabs/TypeScriptSettingTabs.astro

```diff
@@ -0,0 +1,19 @@
+---
+import Tabs from './Tabs';
+---
+
+<Tabs client:visible sharedStore="typescript-settings">
+	<Fragment slot="tab.arelaxed">base ("relaxed")</Fragment>
+	<Fragment slot="tab.bstrict">strict or strictest</Fragment>
+  <Fragment slot="tab.custom">custom</Fragment>
+
+	<Fragment slot="panel.arelaxed">
+		<slot name="relaxed" />
+	</Fragment>
+	<Fragment slot="panel.bstrict">
+		<slot name="strict" />
+	</Fragment>
+  <Fragment slot="panel.custom">
+		<slot name="custom" />
+	</Fragment>
```

Comment:
This solution for ordering works! The other option would probably be numbering them, but `aaaaahhhh` for relaxed would too 😀 

```suggestion
	<Fragment slot="tab.1.relaxed">base ("relaxed")</Fragment>
	<Fragment slot="tab.2.strict">strict or strictest</Fragment>
  <Fragment slot="tab.3.custom">custom</Fragment>

	<Fragment slot="panel.1.relaxed">
		<slot name="relaxed" />
	</Fragment>
	<Fragment slot="panel.2.strict">
		<slot name="strict" />
	</Fragment>
  <Fragment slot="panel.3.custom">
		<slot name="custom" />
	</Fragment>
```

Reply:
This at least makes it clear in the code that the name matters for order, so let's do it! (I was pretty pleased with "aaaahhhh" representing relaxed, though.)